aiida/orm/nodes/data/remote.py

```diff
@@ -57,9 +57,9 @@ def getfile(self, relpath, destpath):
         """
         Connects to the remote folder and gets a string with the (full) content of the file.
 
-        :param relpath: The relative path of the file to show.
-        :param destpath: A path on the local computer to get the file
-        :return: a string with the file content
+        :param relpath:  The relative path of the file on the remote to retrieve.
+        :param destpath: The absolute path of the copied/retrieved file on the local machine.
+        :return: a list containing the file names in 'destpath'
```

Comment:
Indeed, this seems to be what it is doing... 
@giovannipizzi Is there a good reason for doing that (e.g. potentially returning the names of files that have nothing to do with the one that was just transferred?)

Reply:
Just coming back to this - this is actually a bug, right @sphuber ?

This